packages/office-ui-fabric-react/src/components/FocusTrapZone/examples/FocusTrapZone.Box.FocusOnCustomElement.Example.tsx

```diff
@@ -20,51 +21,44 @@ export class FocusTrapZoneBoxCustomElementExample extends React.Component<{}, IF
   private _toggle = React.createRef<IToggle>();
 
   public render() {
+    const { useTrapZone } = this.state;
+
     return (
       <div>
-        {this.state.useTrapZone ? (
-          <FocusTrapZone firstFocusableSelector={_focusClassName}>{this._internalContents()}</FocusTrapZone>
-        ) : (
-          this._internalContents()
-        )}
+        <DefaultButton secondaryText="Focuses inside the FocusTrapZone" onClick={this._onButtonClickHandler} text="Focus Custom Element" />
```

Comment:
Same comment here about maybe replacing the toggle with a second button

Reply:
I think the Toggles keep them consistent with the other examples. The buttons are only needed in two cases to highlight features of the FocusTrapZone (returning focus to initiator and highlighting custom element.)